### PR TITLE
Polymer 2: core fixes

### PR DESCRIPTION
**Pull request by stephanwlee**
This pull request contains three changes that are more platform-y than future pull requests.
Please see individual commit messages for better descriptions of the changes.

### CODE REVIEW

File: tensorboard/components/tf_dashboard_common/data-loader-behavior.ts [hunk 1]
```diff
@@ -139,10 +144,15 @@ export const DataLoaderBehavior = {
...
   },
 
+  _loadDataIfActive() {
+    if (this.active) {
+      this._loadData();
+    }
+  },
+
   _loadDataImpl() {
+    if (!this.active) return;
```
Comment (wchargin) [inline tensorboard/components/tf_dashboard_common/data-loader-behavior.ts]: It doesn’t look like `this.active` is ever set to `true`. What’s
happening here? How does this work?

File: tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-resizer.html [hunk 2]
```diff
@@ -26,20 +26,20 @@
...
-      :host[is-horizontal] {
+      :host([is-horizontal]) {
         cursor: row-resize;
         height: 10px;
         left: 0;
         right: 0;
       }
 
-      :host[_is-vertical] {
+      :host([_is-olvertical]) {
```
Comment (wchargin) [inline tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-resizer.html]: s/olvertical/vertical/

Comment (stephanwlee) [reply tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-resizer.html]: Opps. Done.

File: tensorboard/components/tf_dashboard_common/data-loader-behavior.ts (hunk 1 above)
Comment (stephanwlee) [reply tensorboard/components/tf_dashboard_common/data-loader-behavior.ts]: It is only set by parent (technically component that uses the behavior can hard code it too). 
This should gets used with `tf-category-paginted-view`. i.e.,
https://github.com/stephanwlee/tensorboard/blob/225748afd496f62387682f27b304543282bded37/tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html#L131-L138

Comment (wchargin) [reply tensorboard/components/tf_dashboard_common/data-loader-behavior.ts]: Right, of course. Thanks.

**Pull request merged**